service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/ParticipantMapper.java

```diff
@@ -0,0 +1,39 @@
+package uk.nhs.adaptors.oneoneone.cda.report.mapper;
+
+import java.util.Collections;
+import java.util.List;
+
+import uk.nhs.connect.iucds.cda.ucr.POCDMT000002UK01EncounterParticipant;
+
+import org.hl7.fhir.dstu3.model.CodeableConcept;
+import org.hl7.fhir.dstu3.model.Encounter;
+import org.hl7.fhir.dstu3.model.Practitioner;
+import org.hl7.fhir.dstu3.model.Reference;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ParticipantMapper {
+
+    @Autowired
+    PeriodMapper periodMapper;
```

Comment:
See Java coding standards. Let's try to use constructors instead of field injection.
Field should be private